### PR TITLE
Allow PHP 8

### PR DESCRIPTION
**Pull request by greg0ire**
We want to allow our users to experiment and report bugs if there are
any.

### CODE REVIEW

**Comment by greg0ire** [comment]
Managed to obtain the very same failure as in 7.4… is the current 7.4 build known to be broken?

**Comment by carusogabriel** [comment]
It's broken, `slevomat/coding-standard` fixed a bug and we need to update the expectations.

Also, is this duplicated from https://github.com/doctrine/coding-standard/pull/181?

**Comment by greg0ire** [comment]
I am working on updating the expectations.

This is not a duplicate from #181 : the target branch is not the same, and this does not aim at fixing any issues we might have, or actually supporting php8, it's more of a commitment to support it on the 7.0.x series. This is basically saying that when our dependencies are fixed, we will be fixed.

**Comment by greg0ire** [comment]
Blocked by #193

File: .travis.yml [hunk 1]
```diff
@@ -32,6 +31,11 @@ jobs:
     - php: nightly
 
   include:
+    - stage: Test
+      php: nightly
+      before_install:
+        - composer config platform.php 7.4.99
+
```
Comment (carusogabriel) [inline .travis.yml:38]: Why this instead of allowing `nightly` to fail?

Comment (greg0ire) [reply .travis.yml:38]: It's not "instead", look at line 31, it's already allowed to fail. But now that you fixed the build, I think we are going to get a green build thanks to this trick, which gives us a way better idea of the state of php 8 support in this repository IMO.

**Pull request merged**